examples/torch/maml_ppo_half_cheetah_dir.py

```diff
@@ -1,25 +1,40 @@
 #!/usr/bin/env python3
 """This is an example to train MAML-VPG on HalfCheetahDirEnv environment."""
+# pylint: disable=no-value-for-parameter
```

Comment:
is this solved by providing default args to your launcher function?

Reply:
The default args are provided in `click.option(arg, default=value)`, and later, no arguments is needed to call `maml_ppo()`. But pylint cannot recognize this way in which click handles arguments